satpy/etc/readers/viirs_l2.yaml

```diff
@@ -0,0 +1,126 @@
+reader:
+  name: viirs_l2
+  short_name: VIIRS L2
+  long_name: SNPP VIIRS Level 2 data in netCDF4 format
+  description: Generic NASA VIIRS L2 Reader
+  status: Alpha
+  supports_fsspec: false
+  reader: !!python/name:satpy.readers.yaml_reader.GEOFlippableFileYAMLReader
+  sensors: [viirs]
+  default_datasets:
```

Comment:
This isn't actually used and can be removed. It was something we thought we might use, but never did.

```suggestion
```